placementApp/urls.py

```diff
@@ -6,12 +6,13 @@
 router.register("student_list", views.ListStudentViewSet)
 router.register("positions", views.PositionViewSet)
 
-user_profile = views.UpdateStudentViewSet.as_view(
-    {"get": "retrieve", "put": "update", "patch": "partial_update"}
-)
 
 urlpatterns = [
     path("", include(router.urls)),
     path("get_xls/<int:company_id>", views.get_xls, name="get_xls"),
-    path("student_profile/<int:pk>/", user_profile, name="student_profile"),
+    path(
+        "student_profile/<int:pk>/",
```

Comment:
No need to give <int:pk> in url

Reply:
Changed this locally. Please refer [this comment](https://github.com/djunicode/placement-portal-web/pull/23#discussion_r395820639) and guide me as to what I should do next

Reply:
For now just solve the conflicts and once you have done that check the master branch for all the functionality you need to complete your task

Reply:
Done